### ISSUE TITLE
Visual artefacts in macOS transmission-qt client

### ISSUE THREAD

**Issue by GaryElshaw** [transmission/transmission#4470]
### What is the issue?

Seen at the top of the main window, but also in 'Files' for some reason (screenshot macOS).

![SCR-20221218-q8b](https://user-images.githubusercontent.com/69029666/208283898-9385857a-ecbf-40d5-95cb-f097765b53da.png)

![SCR-20221218-q9p](https://user-images.githubusercontent.com/69029666/208283904-8607f9ac-1803-48d6-ab4b-97314d39a1b1.png)

### Which application of Transmission?

Qt app on Linux, BSD, etc.

### Which version of Transmission?

Beta 2

**Comment by ckerr** [comment]
I'm not seeing this. It it specific to a particular Qt theme?

**Comment by GaryElshaw** [comment]
> I'm not seeing this. It it specific to a particular Qt theme?

It's what i'm seeing on macOS, Manjaro, and Ubuntu LTS. It's also why i know my limits. It's something that needs to be resolved in coding.

**Comment by mikedld** [comment]
Those lines look like misplaced progress bars to me. At least there's no progress bar in the Progress column (where it should be) on the second screenshot, and its width seems to match.

**Comment by GaryElshaw** [comment]
I think that's probable, but note how the top torrent is also split. God knows why that line appears in files.

**Comment by mikedld** [comment]
> It's what i'm seeing on macOS, Manjaro, and Ubuntu LTS.

Please attach screenshots from those other systems as well.

**Comment by GaryElshaw** [comment]
Sorry Mike, i just don't have the energy for that anymore.

**Comment by mikedld** [comment]
Based on your comment, I was under the impression that it's the same issue, so it's important to know if it's Mac-specific or cross-platform. And if it's Mac-specific then the priority will be lower because we don't really test/release/support Qt client on Mac (there's Mac client for that).

**Comment by GaryElshaw** [comment]
It has everything to do with #4376 and me trying my best to get it working, seamlessly, across all platforms.

**Comment by mikedld** [comment]
Incorrect progress bar drawing has nothing to do with icons...

**Comment by GaryElshaw** [comment]
Exactly what i was thinking which is why i thought it was an issue. A. There is no theming on MacOS; and why would a different icon base have this effect?

**Comment by mikedld** [comment]
Looked into it, it's a know issue with Qt on Mac: https://bugreports.qt.io/browse/QTBUG-67830 (see the next to last comment there).

Again, nothing to do with icons. And apparently nothing to do with Linux.

**Comment by GaryElshaw** [comment]
It still doesn't explain the current anomaly of an icon set disturbing the main window, and only the files menu.

**Comment by ckerr** [comment]
> It still doesn't explain the current anomaly of an icon set disturbing the main window, and only the files menu.

I am genuinely a little confused about how (if?) #4470 and #4376 are related? The screenshots in the description here do seem to be showing a clipped progressbar being drawn at the wrong origin, And that's consistent with QTBUG-67830. This ticket doesn't seem to have discussed icons until 40 minutes ago.

**Comment by GaryElshaw** [comment]
@ckerr As i've been working on #4376, different idiosyncrasies have popped up along the way. The changes we made earlier this year for svg's in the qt client have had repercussions where i've now learnt those icons cannot be built unless the qt svg package is installed. Often separately.

I think we've established the icons, per se, are not the issue. Instead we're dealing with theming; and to my mind, consistency across platforms and versions of qt.
